test-specifications/errorCodes.md

```diff
@@ -0,0 +1,37 @@
+# TCK Errors
+
+The JSON-RPC 2.0 specification that the TCK and SDK servers use allow for implementation-specific error codes and server errors. This document specifies all the custom error codes that the TCK uses, what they mean, as well as any additional information the SDK servers should supply to the TCK when sending an error response with these specific error codes.
+
+## Errors
+
+### Hedera Error
+
+#### Error code
+
+`-32001`
+
+#### Usage
+
+When a request is successfully submit to a network, but either does not pass precheck, or does not come to consensus and is rejected. In SDK terms, the `execute()` call for a request is met with a `PrecheckStatusException` or `ReceiptStatusException`.
```

Comment:
nit: Besides the `execute()` call, it can also be `getReceipt()` which can be met with those exceptions.

Reply:
True, though `getReceipt()` calls `execute()` under the hood. I reworded this a bit anyways